aws/awserr.go

```diff
@@ -31,3 +31,24 @@ func retryOnAwsCode(code string, f func() (interface{}, error)) (interface{}, er
 	})
 	return resp, err
 }
+
+func retryOnAwsCodes(codes []string, f func() (interface{}, error)) (interface{}, error) {
```

Comment:
Any ideas on how to best surface these kinds of helper functions for others are much welcome 🤔

Reply:
The only way I can think of is to spend some time on replacing "the old way" with these helpers in all places, b/c most people like to learn from existing code.